devspaces-samples/ansible-devspaces-demo
# feat: use new community image

Replace dev image with new [Community Ansible Dev Spaces Image](https://github.com/ansible/ansible-workspace-env-reference-image/pkgs/container/ansible-workspace-env-reference)

![screenshot-devspaces apps sandbox-m4 g2pi p1 openshiftapps com-2024 04 29-12_16_09](https://github.com/devspaces-samples/ansible-devspaces-demo/assets/1271546/72897b05-82fa-4501-b446-8ef47add0de8)


**Related issue:** https://issues.redhat.com/browse/CRW-6290

**To test current changes:**
1. Login into https://workspaces.openshift.com
2. Start a workspace by clicking the icon below
[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://github.com/devspaces-samples/ansible-devspaces-demo/tree/sv-new-community-image)
3. Check that all functionality works as expected
